Web-Development/NativeJavascript/answers.md

```diff
@@ -0,0 +1,22 @@
+# Answers to the frontend questions Native script
+1. 
+```
+function sum(){
+    let 
+        sumOfArgs=0,
+        i=0;
+        for(;i<arguments.length;i++){
+            sumOfArgs+=arguments[i];
+        }
+    return sumOfArgs
+}
+```
+2. Inheritance works differently in javascript not like in other languages, here we have parent object set in the prototype properties of the object which it refers if it does not find the current object. It will be a chain of references to parent objects in case of multiple inheritances so it is called prototype chain.
+3. Closures in javascript are the block of code and the variables used in it are bound to the block through the variables are defined in its parent object.
+4. `var` variables are hoisted prior to the variables are executed but `const` and `let` or not. And simple properties defined by `const` cannot be changed.
```

Comment:
**All variables** (var, const, let) are hoisted. [Read here](https://stackoverflow.com/a/31222689) Doesn't explain the difference between `let` and `const`. There is no term called "simple properties". Even if you meant that properties owned by the object it's still false.